WhatsNew/WhatsNew/Presentation/WhatsNewView.swift

```diff
@@ -41,6 +41,7 @@ public struct WhatsNewView: View {
                                         .scaledToFit()
                                         .frame(minWidth: 250, maxWidth: 300)
                                         .padding(24)
+                                        .accessibilityIdentifier("whatsnew_image")
```

Comment:
Suggestion to replace with `whats_new_image`

Reply:
whatsnew is actually a feature name and pronounced as single entity.